apps/src/templates/manageStudents/ManageStudentsTable.story.jsx

```diff
@@ -251,100 +253,56 @@ const cleverData = [
   }
 ];
 
-export default storybook => {
-  storybook
-    .storiesOf('ManageStudentsTable', module)
-    .withReduxStore(
+export default {
+  title: 'ManageStudents/ManageStudentsTable',
+  component: UnconnectedManageStudentsTable
+};
+
+const Template = args => (
+  <Provider
```

Comment:
Is the `Provider` needed when we're using the "unconnected" component?

Reply:
I believe the `Provider` is necessary when using redux store, but I may be wrong

Reply:
This story is using the "Unconnected" version of the component, which doesn't use a redux store. Does it break if you remove the Provider stuff? It's possible a subcomponent needs redux, in which case you'll have to leave it in

Reply:
I'm confused about why in first place it was used: 
```
.withReduxStore(
      {manageStudents, teacherSections, unitSelection},
      initialState
    )}
```

Reply:
I think I would much prefer to avoid using the `Provider` wrapping for the storybook, if it is possible. This is the first one I've come across. Just as Bethany suggested: Let's try to remove it and see the component can't be rendered without it. (And leave it in if we must)

Reply:
I interpret this as the redux and provider are necessary.
<img width="1159" alt="Screenshot 2023-04-21 at 11 29 40" src="https://user-images.githubusercontent.com/66776217/233688057-4d7aa5a8-786e-41ae-8827-619830be7f26.png">

Reply:
Ah yeah looks like a subcomponent is using redux then. Unfortunate but not worth a refactor here imo. Thanks for trying!